### ISSUE TITLE
Feature request - Prediction snapshotting/ghost trail

### ISSUE THREAD

**Issue by Ubast** [NightscoutFoundation/xDrip#464]
It would be extremely useful to be able to hit a button a few times during an insulin vs carbs or correction dosage to track states of change to the prediction curve. Allowing better fine tuning of the carb and insulin sensitivity ratios for each person. Currently we are using phone screenshots.

There wouldn't need to be any historical data. Just a graphical representation. I would be happy to create a graphical mock-up of what this could look like. But am not seeing any options to add a file to this request. Thanks for listening.

**Comment by jamorham** [comment]
yes this is something I've wanted to do for a long time but haven't got around to it yet

**Comment by Ubast** [comment]
Great news. I wonder if some volunteering of data might prove useful in an opt in feature one day. I realise this might be useless as everybody's settings may vary and our bodies certainly do. And then there is the accuracy of carb counting. I find a fair bit of variance even with measured repeats of the exact same processed product. And I still have no idea how I would test carb absorbtion or adjust liver settings.

-------- Original message --------

**Comment by norfordb** [comment]
I'd love to see something like this. As an endurance athlete, my insulin/carb and insulin/mg/dL blood sugar ratios change by as much as a factor of 5 depending on what I'm doing.
